community/import-tool/src/test/java/org/neo4j/tooling/ImportToolTest.java

```diff
@@ -920,12 +945,22 @@ private void writeNodeData( PrintStream writer, Configuration config, List<Strin
         {
             if ( linePredicate.accept( i ) )
             {
-                writer.println( nodeIds.get( i ) + delimiter + randomName() +
-                                delimiter + randomLabels( arrayDelimiter ) );
+                writer.println( getLine( nodeIds.get( i ), delimiter, arrayDelimiter, extraColumns ));
             }
         }
     }
 
+    private String getLine(String nodeId, char delimiter, char arrayDelimiter, int extraColumns) {
+        StringBuilder stringBuilder = new StringBuilder(  nodeId  + delimiter + randomName() +
+                delimiter + randomLabels( arrayDelimiter ));
```

Comment:
A shorter way of saying what I said above would be to say that you should instead have written:

```
StringBuilder stringBuilder = new StringBuilder().append( nodeId ).append( delimiter ).append( randomName() ).append( delimiter ).append( randomLabels( arrayDelimiter ) );
```